test/distributed/org/apache/cassandra/distributed/test/IncRepairCoordinatorErrorTest.java

```diff
@@ -51,7 +51,7 @@ public void errorTest() throws Exception
             cluster.get(3).runOnInstance(() -> {
                 ActiveRepairService.instance.failSession(result.toString(), true);
             });
-            assertThat(cluster.get(1).logs().watchFor("Can't transition endpoints .* to FAILED").getResult()).isNotEmpty();
+            assertThat(cluster.get(1).logs().watchFor("Removing completed session .* with state FINALIZED").getResult()).isNotEmpty();
```

Comment:
nit: Any way we could just call into node 1 and get the number of sessions via `ARS` rather than having to check the logs? If that's brutal, no biggie, but I don't even thing we'd have to change the visibility of anything, and we already have the `UUID`?

Reply:
Talked about this with Caleb - I actually prefer the logs check because it's more explicit than "nothing found in map" - it confirms that the specific action (removal from `ARS.consistent.coordinated`) was taken, rather than the more generic "no data found".